docs/community/content/team/_index.en.md

```diff
@@ -9,55 +9,377 @@ chapter = true
 
-| Name                | GitHub ID               |
-| :------------------ | :---------------------- |
-| Craig L Russell     | craiglrussell, clr      |
-| Hao Cao             | haocao                  |
-| Hongjun Du          | codefairy08             |
-| Hongtao Gao         | hanahmily               |
-| Juan Pan            | tristaZero              |
-| Jun Zhao            | cherrylzhao             |
-| Liang Zhang (Chair) | terrymanu               |
-| QingYang Chen       | beckhampu               |
-| Sheng Wu            | wu-sheng                |
-| vongosling          | vongosling              |
-| Willem Ning Jiang   | WillemJiang             |
-| Xiaoguang Ma        | maxiaoguang64           |
-| Yang Yi             | KomachiSion             |
-| Yonglun Zhang       | tuohai666, zhangyonglun |
-| Kimm King           | kimmking                |
-| Haoran Meng         | menghaoranss            |
+<table>
+    <tr>
+        <td>
+            <a href="https://github.com/craiglrussell" target="_blank">
+                <img src="https://avatars.githubusercontent.com/u/2467238?v=4" width="200px">
+            </a>
+        </td>
+        <td>
+            <a href="https://github.com/haocao" target="_blank">
+                <img src="https://avatars.githubusercontent.com/u/687732?v=4" width="200px">
+            </a>
+        </td>
+        <td>
+            <a href="https://github.com/codefairy08" target="_blank">
+                <img src="https://avatars.githubusercontent.com/u/39583929?v=4" width="200px">
+            </a>
+        </td>
+        <td>
+            <a href="https://github.com/hanahmily" target="_blank">
+                <img src="https://avatars.githubusercontent.com/u/1960537?v=4" width="200px">
+            </a>
+        </td>
+        <td>
+            <a href="https://github.com/tristaZero" target="_blank">
+                <img src="https://avatars.githubusercontent.com/u/27757146?v=4" width="200px">
+            </a>
+        </td>
+        <td>
+            <a href="https://github.com/cherrylzhao" target="_blank">
+                <img src="https://avatars.githubusercontent.com/u/8317649?v=4" width="200px">
+            </a>
+        </td>
+        <td>
+            <a href="https://github.com/terrymanu" target="_blank">
+                <img src="https://avatars.githubusercontent.com/u/5516298?v=4" width="200px">
+            </a>
+        </td>
+    </tr>
+    <tr align="center">
+        <td>
+            <b>Craig L Russell</b>
+        </td>
+        <td>
+            <b>Hao Cao</b>
+        </td>
+        <td>
+            <b>Hongjun Du</b>
+        </td>
+        <td>
+            <b>Hongtao Gao</b>
+        </td>
+        <td>
+            <b>Juan Pan</b>
+        </td>
+        <td>
+            <b>Jun Zhao</b>
+        </td>
+        <td>
+            <b>Liang Zhang (Chair)</b>
+        </td>
+    </tr>
+    <tr>
+        <td>
+            <a href="https://github.com/kimmking" target="_blank">
```

Comment:
It 's wrong, Please change to https://github.com/beckhampu and https://avatars.githubusercontent.com/u/14846369?v=4